server/lib/nl/fulfillment/fulfiller.py

```diff
@@ -233,3 +239,50 @@ def _prune_non_country_special_dc_vars(state: PopulateState):
                              list(dropped_vars))
 
   state.chart_vars_map = pruned_chart_vars_map
+
+
+#
+# Prune topic chart vars for toolformer_rig mode
+#
+def _prune_topic_vars_for_rig(state: PopulateState):
+  # Set of all svs that were detected
+  detected_svs = set([
+      sv for sv in state.uttr.detection.svs_detected.single_sv.svs
```

Comment:
Do we need some threshold here, maybe the default threshold?

Reply:
done